pkg/utils/test/setup_helpers.go

```diff
@@ -213,6 +214,10 @@ func ExtractModuleVersion(moduleName string) (string, error) {
 
 // DeployCRDs deploys the CRDs commonly used in tests.
 func DeployCRDs(ctx context.Context, crdPath string, operatorClient *operatorclient.Clientset, cluster clusters.Cluster) error {
+	if test.IsInstallingCRDsDisabled() {
```

Comment:
Can we perform this check in the `DeployCRDs` caller rather than inside this function? I think it'd be easier to navigate.

Reply:
👍 cf14ea85eda6ae5d26895218b2eb413284fa59ba. PTAL